app/views/users/show.html.erb

```diff
@@ -2,6 +2,15 @@
 
 <%= render @user %>
 
+<% if current_user == @user %>
```

Comment:
1. I think `if @user == current_user` is more readable
2. In future it would be better refactor it with policy. Something like `if allowed_to? :update, @user`